docker-compose.yml

```diff
@@ -126,7 +126,7 @@ services:
         hard: 10032
   postgres:
     <<: *restart_policy
-    image: "postgres:9.6"
+    image: "ghcr.io/getsentry/image-mirror-library-postgres:14"
```

Comment:
why aren't we using the original `postgres` image?

note: iirc, I know that sentry mirrors the original postgres image, but people might ask these stuff

Reply:
Great question, this mirrors the original postgres image but uses an older version of the tagged `postgres:14` image, particularly postgres 14.6. I figured it might be good to use the same image that we use in `getsentry/sentry` dev here for consistency purposes, although it shouldn't make a difference

Reply:
Worth commenting that in the file?

Reply:
done

Reply:
Please don't do this as it will block users in China who are blocked by sanctions or some other reasons and are not able to use Google container hosting service. I fear same may apply for GitHub's. See #445.

Reply:
fair point, will change this